tests/Issues/IfElseInline/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Core\Tests\Issues\IfElseInline\Fixture;
+
+final class Fixture
+{
+    /**
+     * @var string
+     */
+    private const INTERFACE = 'Interface';
+
+    public function run(string $className): string
+    {
+        if (str_contains($className, '\\')) {
+            $shortClassName = (string) Strings::after($className, '\\', -1);
+        } else {
+            $shortClassName = $className;
+        }
+
+        return $shortClassName;
+    }
+}
+
+?>
+-----
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Core\Tests\Issues\IfElseInline\Fixture;
+
+final class Fixture
+{
+    /** @var string */
+    private const INTERFACE = 'Interface';
+
+    public function run(string $className): string
+    {
+        $shortClassName = str_contains($className, '\\') ? (string) Strings::after($className, '\\', -1) : $className;
```

Comment:
make ternary here is actually fine as after static call `Strings::after` imported with `importNames()` it got shorter to `109 chars`